erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.py

```diff
@@ -127,17 +161,16 @@ def validate_dates_with_periodicity(self):
 			if d.start_date and d.end_date and d.periodicity and d.periodicity!="Random":
 				date_diff = (getdate(d.end_date) - getdate(d.start_date)).days + 1
 				days_in_period = {
-					"Weekly": 7,
-					"Monthly": 30,
-					"Quarterly": 90,
-					"Half Yearly": 180,
-					"Yearly": 365
+						"Weekly": 7,
+						"Monthly": 30,
+						"Quarterly": 90,
+						"Half Yearly": 180,
+						"Yearly": 365
```

Comment:
over indentation, please scan the files for these, its hard to point every instance of over-indentation here